specification/patient-demographics-service-api.yaml

```diff
@@ -201,6 +202,7 @@ paths:
         Alternatively, you can try out the sandbox using our Postman collection:
 
         [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/2871d49488699f3f5a67)
+        # APM-311 raised for response codes
```

Comment:
As a standard, can we please refrain from using //Comments being commited into the documentation. As this will be rendered as a header.

Any references to specific JIRA tickets, can we add as a comment on the the APM-100 ticket.